stake-pool/cli/src/main.rs

```diff
@@ -1718,7 +1718,7 @@ fn command_withdraw_sol(
 fn command_set_manager(
     config: &Config,
     stake_pool_address: &Pubkey,
-    new_manager: &Option<Keypair>,
+    new_manager: Box<dyn Signer>,
```

Comment:
Can we make this an `Option<Box<dyn Signer>>`? This command allows you to set either the manager or the fee receiver, but the underlying instruction updates both of them at once, which means that if someone only updates the fee receiver, we need to fetch and set the old value.

Reply:
done. would have been cleaner if `get_signer` returned an option or a result type so i just wrapped the call lol